test/e2e/smoketest.go

```diff
@@ -83,9 +114,17 @@ func executeSmokeTest(apiTracesEndpoint, collectorEndpoint string) {
 		Finish()
 	closer.Close()
 
+	transport := &http.Transport{}
+	if hasInsecureEndpoint {
+		insecure := true
+		// #nosec  G402: TLS InsecureSkipVerify set true
+		transport.TLSClientConfig = &tls.Config{InsecureSkipVerify: insecure}
```

Comment:
Same comment as before: we should try to get the service-ca trusted by the client instead of skipping the cert verification.

Reply:
Agreed.  Unfortunately I don't really know how to do this, and my initial attempts to get something working have failed.  So any tips or pointers to appropriate doc would be appreciated.

Reply:
There are a couple of options to get the `service-ca`: 

1. if your test is running inside the cluster (I don't think it is), you can just get the file at `/var/run/secrets/kubernetes.io/serviceaccount/service-ca.crt ` and use that as the CA.
1. if the test is outside of the cluster, you can create a secret annotated with `service.beta.openshift.io/inject-cabundle=true`, and the CA bundle will be at the key `service-ca.crt`.
1. you can also extract the bundle from the existing operator deployment, using `kubectl cp`. But this should be the last resort.

Options 1 and 2 are described here: https://docs.openshift.com/container-platform/4.3/authentication/certificates/service-serving-certificate.html

Reply:
@jpkrohling Wondering if this should be tackled in a separate PR? So initially see whether this approach of moving away from port forwarding (as much as possible) improves the reliability of the tests?

Reply:
Absolutely, as long as there's an issue to track this

Reply:
Ok great, on that basis are you ok with the PR as is?

Reply:
LGTM